src/TunNetCom.SilkRoadErp.Sales.Contracts/Providers/ProviderResponse.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Text.Json.Serialization;
+
+namespace TunNetCom.SilkRoadErp.Sales.Api.Contracts.Providers;
+
+public class ProviderResponse
+{
+    [JsonPropertyName("id")]
+
+    public int Id { get; set; }
+
+    [JsonPropertyName("nom")]
+    public string Nom { get; set; } = null!;
+
+    [JsonPropertyName("tel")]
+    public string Tel { get; set; } = null!;
+    [JsonPropertyName("fax")]
+    public string? Fax { get; set; }
+
+    [JsonPropertyName("matricule")]
+    public string? Matricule { get; set; }
+    [JsonPropertyName("code")]
+    public string? Code { get; set; }
+    [JsonPropertyName("codecat")]
```

Comment:
we should make blank lines between propreties, example:

[JsonPropertyName("matricule")]
public string? Matricule { get; set; }

 [JsonPropertyName("mail")]
 public string? Mail { get; set; }